src/nodes.coffee

```diff
@@ -1252,7 +1252,8 @@ exports.ImportDeclaration = class ImportDeclaration extends ModuleDeclaration
     code.push @makeCode "#{@tab}import "
 
     if @clause? and @clause.length isnt 0
-      for subclause in @clause
+      for subclause, index in @clause
+        code.push @makeCode ', ' if index isnt 0
         code = code.concat subclause.compileNode o
```

Comment:
Isn't it somewhat strange to use an array for `@clause` when it can only contain one or two values? This code kinda implies that `@clause` could be of any length. I guess it doesn't matter in the end though…